src/test/java/com/exasol/dbbuilder/ConnectionDefinitionTest.java

```diff
@@ -0,0 +1,99 @@
+package com.exasol.dbbuilder;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import com.exasol.dbbuilder.objectwriter.DatabaseObjectWriter;
+
+@ExtendWith(MockitoExtension.class)
+class ConnectionDefinitionTest {
+    private static final String CONNECTION_NAME = "CONNECTIONNAME";
+    @Mock
+    private DatabaseObjectWriter writerMock;
+
+    @Test
+    void testGetName() {
+        assertThat(new ConnectionDefinition(this.writerMock, CONNECTION_NAME, "IRRELEVANT").getName(),
```

Comment:
Extract connection to (definition, targets, destination) to constant variable.

Reply:
Can't extract to static constant since `writerMock` is an instance variable. I could extract it into `beforeEach`, but then I create it unnecessarily for test cases where I mutate the parameters.

Reply:
Ahh okay, makes sense.

Reply:
Sorry, I think I meant here the `IRRELEVANT` destination part. It is also not possible to extract to variable? For example, to `CONNECTION_DEST` or `CONNECTION_TARGET`.

Reply:
Now I understand. Done.